src/rstxml2db/rstxml2db.xsl

```diff
@@ -639,14 +639,25 @@
         <xsl:with-param name="filename" select="@uri"></xsl:with-param>
       </xsl:call-template>
     </xsl:variable>
-    <mediaobject>
+    <xsl:variable name="imagedata">
+     <imagedata fileref="{$uri}">
+      <xsl:if test="@width">
+       <xsl:attribute name="width">
+        <xsl:value-of select="@width"/>
```

Comment:
Maybe make sure that this is always a percentage and that that percentage is always <= 100%..?
This is more of a nit (that creates a bunch of work), though. Otherwise, it looks fine.